universal-login-react/src/config/index.ts

```diff
@@ -1,11 +1,17 @@
+import {ReactConfig} from './types';
+
 require('dotenv').config();
 import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
 
-const developmentConfig = {
+const developmentConfig: ReactConfig = {
```

Comment:
I am not sure if `ReactConfig` is proper names. Sounds like general React config. Not like Universal Login React Components config. Perhaps `ULReactConfig`?

Reply:
Took inspiration from `SdkConfig`. I'll rename `ReactConfig` -> `Config` for now, the config will surely be revisited in the future.